pkg/execer/client.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package execer
```

Comment:
Execer gives you a struct that wraps some exec stuff. Basically, you get an object that lets you exec things and pipe files to it and returns the stdout/stderr if wanted.